src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+  align-items: center;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background: url(../images/star.svg) center no-repeat;
+
+    & + & {
+      margin-left: 4px;
```

Comment:
The margin-left value should also be defined as a variable if it's a commonly used value within your styles to ensure consistency and ease of future adjustments.